app/models/user.rb

```diff
@@ -14,14 +14,15 @@ class User < ActiveRecord::Base
     :twitter_username
   ].freeze
 
+  before_destroy :yank_gems
```

Comment:
I agreed, deleting a user should not yank all their gems if there are multiple owners.